lib/src/index.ts

```diff
@@ -15,3 +15,4 @@ export * from "./components/TextArea";
 export * from "./components/Tab";
 export * from "./components/Avatar";
 export * from "./components/Table";
+export * from "./components/ComboBox";
```

Comment:
we decided to change name to Select right?

Reply:
forgot to rename the folder